hardware_interface/include/hardware_interface/hardware_info.hpp

```diff
@@ -126,6 +126,32 @@ struct TransmissionInfo
   std::unordered_map<std::string, std::string> parameters;
 };
 
+/**
+ * This structure stores information about an interface for a specific hardware which should be
+ * instantiated internally.
+ */
+struct InterfaceDescription
+{
+  InterfaceDescription(const std::string & prefix_name_in, const InterfaceInfo & interface_info_in)
+  : prefix_name(prefix_name_in), interface_info(interface_info_in)
+  {
+  }
```

Comment:
Why do we need constructor here? Don't `struct` has default constructor?

Reply:
It has the initializer list approach where you can forget to define parts, not with this.